<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-3OO38O2" ignored="0" name="Default test" tests="3" failures="0" timestamp="2022-12-11T14:45:27 CST" time="0.059" errors="0">
  <testcase name="@BeforeMethod init" time="0.004" classname="test.AddCategoryTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at util.Browserfactory.init(Browserfactory.java:35)
at test.AddCategoryTest.init(AddCategoryTest.java:28)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:63)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:348)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:302)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:695)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:523)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeMethod init -->
  <testcase name="@BeforeMethod init" time="-1.670791527407E9" classname="test.AddCategoryTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod init -->
  <testcase name="@BeforeMethod init" time="-1.670791527411E9" classname="test.AddCategoryTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod init -->
  <testcase name="validateAddCategory" time="0.0" classname="test.AddCategoryTest">
    <skipped/>
  </testcase> <!-- validateAddCategory -->
  <testcase name="dublicateEntry" time="0.0" classname="test.AddCategoryTest">
    <skipped/>
  </testcase> <!-- dublicateEntry -->
  <testcase name="monthDropDown" time="0.0" classname="test.AddCategoryTest">
    <skipped/>
  </testcase> <!-- monthDropDown -->
</testsuite> <!-- Default test -->
